tests/test_storage.py

```diff
@@ -648,45 +701,61 @@ def test_client_without_credentials(self):
 class TestAzureStorage(unittest.TestCase):
     """Tests for storing packages in Azure Blob Storage"""
 
-    @classmethod
-    def setUpClass(cls):
-        super(TestAzureStorage, cls).setUpClass()
-        if sys.version_info.major < 3 or sys.version_info.minor < 6:
-            raise unittest.SkipTest("vcrpy does not work on < 3.6 :(")
-
     def setUp(self):
         super(TestAzureStorage, self).setUp()
         self.settings = {
             "pypi.storage": "azure-blob",
-            "storage.storage_account_name": "terrytest2",
-            "storage.storage_account_key": "Aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa==",
+            "storage.storage_account_name": "devstoreaccount1",
+            "storage.storage_account_key": "Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==",  # https://github.com/Azure/Azurite#default-storage-account
+            "storage.storage_account_url": "http://devstoreaccount1.azurite:10000",
             "storage.storage_container_name": "pypi",
         }
         storage_func = get_storage_impl(self.settings)
         self.storage = storage_func(MagicMock())
+        try:
+            self.storage.container_client.create_container()
+        except ResourceExistsError:
+            # https://github.com/Azure/Azure-Functions/issues/2166#issuecomment-1159361162
+            pass
```

Comment:
Would it be possible to skip these tests if azurite isn't available, like we do here? https://github.com/stevearc/pypicloud/blob/9681ca6091a5409855c8f8472be9ebb29634c384/tests/test_cache.py#L194-L197

At the moment the tests just hang

Reply:
so, with azurite available:
- without explicit `create_container`, downstream code will fail due to missing container
- with explicit `create_container`, this line will (erroneously) throw `ResourceExistsError`

so we need to create the container, but ignore the error that's thrown when the container gets created 🤯

Reply:
ah, read again. you mean the scenario where the azurite container isn't started. let me see

Reply:
[396f7ac](https://github.com/stevearc/pypicloud/pull/304/commits/396f7acfd1effdf2d0c8267c0250a114d6a9926a)